packages/next-auth/src/next/index.ts

```diff
@@ -82,21 +82,29 @@ function NextAuth(
 }
 
 export default NextAuth
+let shown = false
 
 export async function unstable_getServerSession(
```

Comment:
```suggestion

let experimentalWarningShown = false
export async function unstable_getServerSession(
```

Taking advantage of white space to make it more obvious that this flag is only used within this function 💅🏽

Making the name a bit more explicit as well ☕️

Reply:
Dig it :+1: